src/bitmessagemain.py

```diff
@@ -174,16 +181,17 @@ def signal_handler(signum, frame):
     if shared.thisapp.daemon or not state.enableGUI:
         shutdown.doCleanShutdown()
     else:
-        print('# Thread: %s(%d)' % (thread.name, thread.ident))
+        print '# Thread: %s(%d)' % (thread.name, thread.ident)
```

Comment:
I disagree, this is for python3 compatibility. Probably better to suppress the warning.

Reply:
Okay